receptor-python-worker/tests/test_receptor_python_worker.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import receptor_python_worker
+
+
+class TestRun:
```

Comment:
In general, when using `pytest`, classes are overkill. You already have regular modules to namespace different tests. So unless there's a good reason, I tend to just use simple functions.